cpp/include/cudf/column/column_factories.hpp

```diff
@@ -42,13 +42,37 @@ namespace cudf
  * kernels
  * @param[in] mr Optional resource to use for device memory
  * allocation of the column's `data` and `null_mask`.
- *---------------------------------------------------------------------------**/
```

Comment:
I love how @jrhemstad keeps adding horizontal rules and @davidwendt keeps removing 'em. :)

Reply:
It's not something I usually think about. My VSCode is just set up to add them. I've changed the setting now.

Reply:
I'm just teasing you -- I don't care either way. It was not I who complained about the lines in the past...